frontend/src/metabase/query_builder/components/QuestionActions.tsx

```diff
@@ -107,6 +108,7 @@ const QuestionActions = ({
     const modal = checkCanBeModel(question)
       ? MODAL_TYPES.TURN_INTO_DATASET
       : MODAL_TYPES.CAN_NOT_CREATE_MODEL;
+    trackTurnIntoModelClicked(question);
```

Comment:
I asked product people about [whether or not this is the button/menu we want to track](https://metaboat.slack.com/archives/C01LQQ2UW03/p1686300137312269?thread_ts=1686064554.952519&cid=C01LQQ2UW03). So let's wait for the confirmation.

Reply:
Anyhow, even if this is the place we want to track still there should be the following question if we want to track this for both cases
1. When we can actually turn this question into a model
2. When we can't turn this question into a model (as shown in the screenshot below)
![image](https://github.com/metabase/metabase/assets/1937582/cba57c0f-f490-45d5-be93-78a48d7c34cc)

The second case sounds really odd at least to me. Like we're tracking the menu click even though we know this question couldn't be turned into a model.

Reply:
Ok so we're tracking the menu click. [Context](https://metaboat.slack.com/archives/C01LQQ2UW03/p1686302270894519?thread_ts=1686064554.952519&cid=C01LQQ2UW03).